### ISSUE TITLE
Improve components used for <Suspense /> wrappers

### ISSUE THREAD

**Issue by jeffstieler** [woocommerce/woocommerce#32281]
**Is your feature request related to a problem? Please describe.**

Lazy loading was introduced in https://github.com/woocommerce/woocommerce-admin/pull/4094, but only uses basic `<Spinner />`s for the loading components.

**Describe the solution you'd like**

Ideally, we could use the placeholder components already present in our app for waiting on network requests.

**Should this be prioritized? Why?**

I believe so - users on slower connections will likely see these spinners the first time they load our newly chunked JS files.

**Comment by rjchow** [comment]
Some improvement on the cohesiveness and fluidity of our frontend user interface could improve the perceived quality of woocommerce-admin by users, so I believe there is long term value in this piece of work

Started preliminary scoping on this at p90Yrv-2FJ-p2

**Comment by chihsuan** [comment]
I think we need to split this into smaller pieces to work on.

@rjchow As you summarized in the P2, would you like to create some tickets for them?

> In summary, for the portion of the application that was covered in this Part 1, I have only two recommendations:
> - Create a suitable placeholder with animated transition for the <Spinner> in client/layout/controller.js
> - Use <Spinner> for the loading of themes thumbnails

**Comment by rjchow** [comment]
I think most of the issues raised in the P2 are no longer relevant as they are either deprecated (old OBW), or already have spinners added. Hence I'm closing this issue
